backend/app/api/routes/attendance.py

```
@@ -1,38 +1,184 @@
-        # TODO: implement real face recognition here
-        # img.save("last_capture.jpg")  # example
-    except Exception as e:
-        return {"error": str(e)}
-
-    # TEMP stub (same as your old code)
-    detected = [
-        {"roll": "2101", "name": "Ravi Kumar"},
-        {"roll": "2122", "name": "Mira Singh"},
-    ]
+        image_bytes = base64.b64decode(image_b64)
+    except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)